.github/workflows/release-python.yml

```
@@ -0,0 +1,293 @@
+          limit-severities-for-sarif: true
+          exit-code: '1'
+
+      - name: Upload Trivy scan results to GitHub Security tab
+        uses: github/codeql-action/upload-sarif@v4
+        if: always()
+        with:
+          sarif_file: 'trivy-results.sarif'
+
+  finalize:
+    needs: [validate, pypi-publish, docker]
+    if: |
```

actionlint (1.7.10)

[error] if: condition "${{ !cancelled()\n  && needs.validate.result == 'success'\n  && needs.pypi-publish.result == 'success'\n  && (needs.docker.result == 'success'\n    || needs.docker.result == 'skipped') }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)